risc0/zkvm/README.md

```diff
@@ -27,3 +27,25 @@ channel](https://www.youtube.com/@risczero) has many more videos as well.
   see if there's a workaround, and if you're using a workaround, you can track
   when it gets resolved to a permanent solution.
 * And more on [the RISC Zero website](https://www.risczero.com/)!
+
+# Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
+
+* cuda
+  * Optional (default is CPU only)
+  * Turns on CUDA GPU acceleration for the prover
+* metal
+  * Optional (default is CPU only)
+  * Turns on Metal GPU acceleration for the prover
+  * Only available for MacOS
+* profiler
+  * Optional
+  * Use with `r0vm` binary tool to profile guest code
+  * Counts cycles during guest execution as an aid to code optimization
+* prove
+  * Enabled by default
+  * Disable this feature if you only need to use the verifier, or if you want to use the `guest` module in guest-side code
```

Comment:
I would flip the order on this to highlight the more important "zkvm in guest" case first: Using `guest::env` in the guest shows up in literally all of our examples. Running the verifier w/out the prover is another use, and I do think it's worth mentioning, but it's not even strictly necessary to exclude the prover in that case (you just build unnecessary stuff). Whereas the build fails if you try to make guest code depend on `risc0_zkvm` without disabling the `prove` feature.

I'd make the analogous change in the zkp readme as well.

Reply:
Tables added to top-level README and to all relevant crate-level READMEs.
Language updated for "zkvm in guest" case and order flipped in all crate notes.